Papr/Scenes/Search/SearchPhotos/Cell/SearchPhotosCellModel.swift

```diff
@@ -8,8 +8,11 @@
 
 import Foundation
 import RxSwift
+import UIKit
 
-protocol SearchPhotosCellModelInput {}
+protocol SearchPhotosCellModelInput {
+    var photoSize: Variable<CGSize> { get }
+}
 protocol SearchPhotosCellModelOutput {
     var smallPhotoURL: Observable<String> { get }
     var regularPhotoURL: Observable<String> { get }
```

Comment:
So, the update function in input for the viewModel (its's coming form the viewController). This input will give us an output, which is the photo size, which later we will bind to `updateSize` func. Based on this, create an observable
`var photoSize<(width: Double, height: Double)>`, in the output `protocol`.